package.json

```diff
@@ -43,6 +43,8 @@
     "axe-webdriverjs": "0.4.0",
     "babel-core": "6",
     "babel-loader": "6",
+    "babel-runtime": "^6.26.0",
```

Comment:
Should we put this under dependencies? https://babeljs.io/docs/plugins/transform-runtime/